File: checkVowel.py
def is_vowel_present(string):
    vowels = "aeiouAEIOU"
    
    for char in string:
        if char in vowels:
            return True
    
    return False

# Example usage:
input_string = input("Enter a string: ")
if is_vowel_present(input_string):
    print("Vowel is present in the string.")
else:
    print("No vowel is present in the string.")

File: checkVowel.js
function isVowelPresent(str) {
    const lowerCaseStr = str.toLowerCase();
    const vowels = ['a', 'e', 'i', 'o', 'u'];

    for (let i = 0; i < lowerCaseStr.length; i++) {
      if (vowels.includes(lowerCaseStr[i])) {
        return true; 
      }
    }
  
    return false;
  }
  
  const inputString = "Kavisha Gupta";
  const result = isVowelPresent(inputString);
  
  if (result) {
    console.log("The string contains at least one vowel.");
  } else {
    console.log("The string does not contain any vowels.");
  }
  